f5-bigip-mixin/config.libsonnet

```diff
@@ -0,0 +1,21 @@
+{
+  _config+:: {
+    dashboardTags: ['f5-bigip-mixin'],
+    dashboardPeriod: 'now-1h',
+    dashboardTimezone: 'default',
+    dashboardRefresh: '1m',
+
+    grafanaLogDashboardIDs: {
+      'f5-logs-overview.json': 'f5-logs-overview',
+    },
+
+    // alerts thresholds
+    alertsCriticalNodeAvailability: 95,  // %
+    alertsWarningServerSideConnectionLimit: 80,  // %
+    alertsCriticalHighRequestRate: 150,  // %
```

Comment:
is this even possible? how would the requests over the last 10 minutes / requests over the last 60 minutes (which includes that 10 min windows) ever be greater than 100%?

Reply:
This is not possible with an increase function since the 60 minutes includes the last 10 minutes and therefore the numerator can never be higher than the denominator and be greater than 100 (assuming no negatives). There was some confusion going back and fourth between rate and increase here. One way to fix this while still using an increase is by using an offset `increase(bigip_pool_tot_requests[50m] offset 10m)` . I'll simulate some load tomorrow to get a better picture of this.

Reply:
testing this out more with some load, rate is the only way forward with this approach. updated to rate.